src/test/resources/features/refapp-2.x/03-trackPatient/vitalsAndTriaging.feature

```diff
@@ -0,0 +1,12 @@
+Feature: Vitals Management
+
+  Background:
+    Given a user clicks on Capture Vitals link from Patient dashboard
+    Then the system loads Vitals page
+
+  @selenium
+  @vitals
+  Scenario: Capture Patient vitals
```

Comment:
Please add extra scenarios to handle success/failure and validation cases

Reply:
> add extra scenarios to handle success/failure and validation cases

a little more light on this and then I act accordingly

Reply:
Christine will be editing the description to provide more context on this, you can reach-out to her for more feedback